docs/shard.yml.schema.json

```diff
@@ -0,0 +1,255 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "shard.yml",
+  "description": "Metadata for projects managed by Crystal Shards",
```

Comment:
**question:** Why *Crystal Shards*? The tool is called just *Shards*.

Reply:
I'm fine either way. Sija recommended "Crystal Shards" and I accepted it.